File: week03/src/main/kotlin/chapter2/expression.java
package chapter2;

public class expression {
    private static String name = "gmin";

    public static void main(String[] args) {
        // 자바는 코틀린과 달리 if가 문이여서, 값이 나오지 않음.
        if (name instanceof String) {
            System.out.println(name);
        }
        System.out.println(name = "❤서원❤"); // 자바는 코틀린과 달리, 대입이 식임.
    }
}

File: week02/src/main/kotlin/chapter2/expression.kt
package kos.chapter2

val name : String = "gmin"

fun main() {
    val result = if (name is String) name else "이름 없음"
    // if문이 식임을 보기 위해 억지로 작성된 코드임.
    // 현실적으로는 val result = name ?: "이름 없음"
    println(result)

    // val rename = result()
}

